test/unit/core/test_group.py

```diff
@@ -27,14 +26,24 @@ def setUp(self):
         self.cluster = demo.new_cluster(demo.generate_inventory(**demo.FULLHA), fake=False)
 
     def test_nodegroup_result_to_str(self):
-        expected_results_str = '''	10.101.1.2: code=1
-		STDERR: sudo: kubectl: command not found
-	10.101.1.3: code=1
-		STDERR: sudo: kubectl: command not found
-	10.101.1.4: code=1
-		STDERR: sudo: kubectl: command not found'''
+        expected_results_str = ("""\
+            10.101.1.2: code=1
+            \t=== stderr ===
+            \tsudo: kubectl: command not found
+            \t
+            10.101.1.3: code=1
+            \t=== stderr ===
+            \tsudo: kubectl: command not found
+            \t
+            10.101.1.4: code=1
+            \t=== stderr ===
+            \tsudo: kubectl: command not found
+            \t"""
```

Comment:
I changed logging format of commands results. Previously it was
```
	10.102.0.133: code=0
		STDOUT: line 1
		        line 2
```
Now it is
```
10.101.17.41: code=0
	=== stdout ===
	line 1
	line 2
```
This is to make it similar to the format that is printed if UnexpectedExit occurs where `str(invoke.Result)` is typically printed. See https://github.com/pyinvoke/invoke/blob/1.6.0/invoke/runners.py#L1473

Please be free to comment on this and ask me to return the previous format if you find it more pretty.